test/pg_regress.sh

```diff
@@ -22,6 +18,11 @@ TESTS=${TESTS:-}
 IGNORES=${IGNORES:-}
 SKIPS=${SKIPS:-}
 
+if [[ -z "${COMMAND}" ]];  then
```

Comment:
Suggestion: braces are not normally needed for variable, only in special cases, and quotes are not needed for conditional expressions (`[[`) only when using `test` (that is`[`).
```suggestion
if [[ -z $COMMAND ]];  then
```